lib/auth/resource/authentication.go

```diff
@@ -14,170 +14,19 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package types contains all types and logic required by the Teleport API.
-
-package services
+package resource
 
 import (
-	"bytes"
 	"encoding/json"
 	"fmt"
-	"time"
 
 	"github.com/gravitational/teleport/api/types"
-	"github.com/gravitational/teleport/lib/auth/u2f"
+	"github.com/gravitational/teleport/lib/auth"
 	"github.com/gravitational/teleport/lib/utils"
 
 	"github.com/gravitational/trace"
-	"golang.org/x/crypto/bcrypt"
 )
 
-// ValidateLocalAuthSecrets validates local auth secret members.
```

Comment:
Moved to [lib/auth/authentication.go](https://github.com/gravitational/teleport/blob/dmitri/split-client-server/lib/auth/authentication.go#L34)